samples/benchmarks/coremark/prj.conf

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Common coremark configuration
+CONFIG_COREMARK=y
+
+CONFIG_COREMARK_ITERATIONS=2000
+
+# Choose proper memory size based on memory method selected above.
```

Comment:
Consider explicitly defining memory method (otherwise comments refers to something that does not actually exist)

Reply:
removed